3_javascript/javascript2/week1/script.js

```diff
@@ -45,30 +41,36 @@ const spiritNames = [
     "The conniving monkey",
 ];
 
+function getRandomSpiritName() {
+    const randomSpirits = Math.floor(Math.random() * spiritNames.length);
```

Comment:
nice! to make it even neater you could put the spirit animal array into the function, since the array is not used anywhere else. As a rule of thumb it's best to not rely on external scope inside the functions if possible. I will approve though, it's good work!